src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -291,6 +293,32 @@ impl HummockEventHandler {
         // Notify completion of the Clear event.
         notifier.send(()).unwrap();
     }
+
+    fn handle_version_update(&self, version_payload: Payload) {
+        let max_committed_epoch_before_update =
+            self.local_version_manager.get_pinned_version().id();
+
+        if let Some(new_version) = self
+            .local_version_manager
+            .try_update_pinned_version(version_payload)
+        {
+            let new_version_id = new_version.id();
+            let max_committed_epoch_after_update = new_version.max_committed_epoch();
+
+            if max_committed_epoch_before_update != max_committed_epoch_after_update {
+                self.read_version
+                    .write()
+                    .update(VersionUpdate::CommittedSnapshot(new_version));
+
+                self.local_version_manager
+                    .notify_version_id_to_worker_context(new_version_id);
+            }
```

Comment:
It makes sense to not notify local version manager if MCE doesn't change but shouldn't we update read version no matter whether MCE changes?

Reply:
yes， we should update read version no matter whether MCE changes?